homeassistant/components/azure_servicebus/notify.py

```diff
@@ -0,0 +1,83 @@
+"""Support for azure service bus notification."""
+import json
+import logging
+
+from azure.servicebus.aio import Message, ServiceBusClient
+import voluptuous as vol
+
+from homeassistant.components.notify import (
+    ATTR_DATA,
+    ATTR_MESSAGE,
+    ATTR_TARGET,
+    ATTR_TITLE,
+    PLATFORM_SCHEMA,
+    BaseNotificationService,
+)
+from homeassistant.const import CONTENT_TYPE_JSON
+import homeassistant.helpers.config_validation as cv
+
+CONF_CONNECTION_STRING = "connection_string"
+CONF_QUEUE_NAME = "queue"
+CONF_TOPIC_NAME = "topic"
+
+PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
+    {
+        vol.Required(CONF_CONNECTION_STRING): cv.string,
+        vol.Exclusive(
+            CONF_QUEUE_NAME, "output", "Can only send to a queue or a topic."
+        ): cv.string,
+        vol.Exclusive(
+            CONF_TOPIC_NAME, "output", "Can only send to a queue or a topic."
+        ): cv.string,
+    }
+)
+
+_LOGGER = logging.getLogger(__name__)
+
+
+def get_service(hass, config, discovery_info=None):
+    """Get the notification service."""
+    connection_string = config[CONF_CONNECTION_STRING]
+    queue_name = config.get(CONF_QUEUE_NAME)
+    topic_name = config.get(CONF_TOPIC_NAME)
+
+    servicebus = ServiceBusClient.from_connection_string(
+        connection_string, loop=hass.loop
+    )
+
+    if queue_name:
+        client = servicebus.get_queue(queue_name)
+    elif topic_name:
+        client = servicebus.get_topic(topic_name)
+    else:
+        _LOGGER.error(
+            "Could not find a topic or queue from the configuration. Please provide one!"
```

Comment:
Please add `cv.has_at_least_one_key` in the config schema instead.

Reply:
Didn't know about that one, thanks for enlightening me! Have updated the configuration 😃